apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -60,6 +60,9 @@ type FlowCollectorSpec struct {
 	// `loki`, the flow store, client settings.
 	Loki FlowCollectorLoki `json:"loki,omitempty"`
 
+	// `prometheusClient` defines Prometheus client settings, used to fetch metrics from the Console plugin.
+	PrometheusClient FlowCollectorPrometheusClient `json:"prometheusClient,omitempty"`
+
```

Comment:
```suggestion
	// `prometheus` defines Prometheus client settings, used to fetch metrics from the Console plugin.
	Prometheus FlowCollectorPrometheus `json:"prometheus,omitempty"`
```

For consistency, I feel it would be better to simply have `prometheus` as same as `loki`

Reply:
that could contains FLP prometheus settings in future

Reply:
That makes sense indeed. And we should also end up allowing users to disable prometheus export from FLP too indeed

Reply:
the current way to do this would be to set an empty `includeList` of metrics